benchmarks-runner/src/main/kotlin/runners/hashMapBuilderRunner.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2016-2019 JetBrains s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package runners
+
+import hashMapBuilderOutputFileName
+import implementationParam
+import runBenchmarks
+
+
+fun main(args: Array<String>) {
```

Comment:
Minor: args are not used in runners so they can be omitted.

Reply:
done